documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,161 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add Open Virtual Appliance (OVA) files that were created by or are compatible with VMware vSphere as source providers.
+
+[IMPORTANT]
+====
+Migration using Open Virtual Application (OVA) files that were created by or are compatible with VMware vSphere as source providers is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during the development process.
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::[]
```

Comment:
I suppose #410 would be a better place for this comment but we should have a `.Prerequisites` section that asks the user to place the OVAs on an NFS share

Reply:
@ahadas We have a prerequisites section for each of the types of source providers before the installing the Operator section. So  that's where I've put the OVA prerequisites we discussed. Would you like me to add a prerequisites section for OVA here? Most of the source providers don't have prerequisites in the "Create providers" section.

Reply:
ack